bolt-modules/ctrl/spec/functions/ctrl/do_until_spec.rb

```diff
@@ -3,13 +3,32 @@
 require 'spec_helper'
 
 describe 'ctrl::do_until' do
-  it {
-    count = 0
-    seq = ['something', false, false]
+  let(:seq) { [false, 'truthy', false, false] }
```

Comment:
Should 'truthy' be the last element here, assuming it's ordered? I don't think the 'exits early with limit 2' test will actually test exiting early with this order.